phpunit.xml.dist

```diff
@@ -11,16 +11,15 @@
   <php>
     <ini name="memory_limit" value="-1"/>
   </php>
-
   <testsuites>
     <testsuite name="Roave\InfectionStaticAnalysis tests">
       <directory>./test/unit</directory>
     </testsuite>
   </testsuites>
-
-  <coverage>
+  <coverage/>
```

Comment:
IMO it's not needed, as `<source>` is used by default. Or is it bring any other sense here? Would like to know as at Infection we just made an update related to this PHPunit change, so if it has some meaning here, would like to understand what :)

Reply:
@maks-rafalko I've used PHPUnit `--migrate-configuration`

Reply:
thanks for letting me know, though not clear 😃